packages/net/hass-tracker/files/functions.sh

```diff
@@ -21,21 +21,24 @@ post() {
         exit 1
     fi
     payload=$1
-    
+
     config_get hass_host global host
     config_get hass_token global token
-    
-    resp=$(curl "$hass_host/api/services/device_tracker/see" -sfSX POST \
+    config_get hass_curl_insecure global curl_insecure
+
+    [ -n "$hass_curl_insecure" ] && curl_param="-k"
```

Comment:
if `hass_curl_insecure=0` this would still add the parameter, contrary to expectation